tutorials/profiler.md

```diff
@@ -73,6 +72,12 @@ cmake ..
 make profiler_example
 ```
 
+Or if you are building with ``colcon``, enable profiler by passing in ``cmake`` args.
```

Comment:
Is this necessary on this example even though `IGN_PROFILER_ENABLE` is already set on the CMake file above?

I think that your suggestion is important when compiling the Ignition libraries, because unlike this example, they have the profiler [disabled by default](https://github.com/ignitionrobotics/ign-gazebo/blob/8ec8d66cc8a6b661a2db6ccae153a785287fa315/CMakeLists.txt#L23-L29). How about explaining that instead? Something like:

```
If you want to use the profiler of an Ignition library, be sure to enable
the profiler at compile time with the `ENABLE_PROFILER` CMake
argument.

When using plain CMake:

`cmake .. -DENABLE_PROFILER=1`

When using colcon:

`colcon build --cmake-args -DENABLE_PROFILER=1`
```
I think this explanation could be outside of this example though.